packaging/templates/mainnet-v1/sentry/sentry/bor/pbss_config.toml

```diff
@@ -24,7 +25,7 @@ gcmode = "archive"
     # enable-block-tracking = false
 
 [p2p]
-    maxpeers = 50
```

Comment:
what reservations do we have setting this value higher than 200 ? for the mesh backbone important we can connect to as many peers as possible for relaying blocks

Reply:
The 200 number was provided based on some testing and good number of peers, these configs are baselines to use and can be modified to any need.